tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -217,11 +217,13 @@ public function testConvertColumn($type, $expected)
             'default' => 'Default value',
             'comment' => 'Comment section',
             'char_length' => null,
+            'collation_name' => 'Collate information',
         ];
         $expected += [
             'null' => true,
             'default' => 'Default value',
             'comment' => 'Comment section',
+            'collate' => 'Collate information',
```

Comment:
Is "Collation name" suitable? I forgot why I chose the word, since I wrote this test case some months ago.